packages/perseus/src/util/parse-perseus-json/perseus-parsers/interactive-graph-widget.ts

```diff
@@ -276,18 +277,23 @@ export const parseInteractiveGraphWidget: Parser<InteractiveGraphWidget> =
         constant("interactive-graph"),
         object({
             step: pairOfNumbers,
-            gridStep: pairOfNumbers,
-            snapStep: pairOfNumbers,
+            gridStep: optional(pairOfNumbers),
+            snapStep: optional(pairOfNumbers),
             backgroundImage: optional(parsePerseusImageBackground),
             markings: enumeration("graph", "grid", "none"),
-            labels: array(string),
+            labels: defaulted(array(string), () => []),
```

Comment:
This default isn't right... `labels` is for the X and Y axis labels, not locked labels. The default labels are currently assigned in the default props of the InteractiveGraph component.

I think the correct fix is to let `labels` be `optional` here and in the `PerseusInteractiveGraphWidgetOptions` type. The props type for InteractiveGraph should not use `PerseusInteractiveGraphWidgetOptions`.